gc/verbose/schema.xsd

```diff
@@ -612,6 +612,7 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 	<complexType name="concurrent-end">
 		<sequence>
 			<element ref="vgc:concurrent-mark-end" maxOccurs="1" minOccurs="1" />
+			<element ref="vgc:warning" maxOccurs="unbounded" minOccurs="0" />
```

Comment:
slight nitpick. Can there be more than warning per concurrent GC end?

Reply:
yes, there could be a gc-op. clearly, missed from a long ago

Reply:
added

Reply:
This particular warning could show up only once, but since I'm reusing generic warning message, perhaps there could be some other ones too (overflow etc), although I'm not sure (perhaps others could show up only in the final STW gc-op). I can investigate if that's the case....

For gc-op, well, clearly it's a part of it (see example below), although it does not have to show up in context of Balanced GC (hence, min=0).

Reply:
Thanks for the clarification!